### PR TITLE
Update plugin-pdf.md

### PR DESCRIPTION
**Pull request by candywal**

### CODE REVIEW

File: docs/sma/plugin-pdf.md [hunk 1]
```diff
@@ -166,6 +166,7 @@ Non-hotkeys
 			- Add to selection (Hold ctrl)
 			- Select image (Click image)
 ```
+<p class="tip">⚠️ When you extract text from the PDF file, the windows error sound may play.</p>
```
Comment (alexis-) [inline docs/sma/plugin-pdf.md]: 1) Syntax

Use markdown instead. HTML is reserved for these occasions where we are forced to use it (e.g. for something that is not possible with markdown)

The equivalent for this is:
```md
> [!NOTE]
> When you extract text from the PDF file, the windows error sound may play.
```

2) It is nice to warn the user, but it is even better to provide a solution. Fortunately, you already have one ! Inform the user and link to the FAQ.

Every heading (all the markdown lines that begin with # up to -> ######) has an "id" that acts as an anchor. You can check for the heading's ID by moving your mouse over it in your browser, and checking where it sends you to (the url that appears at the bottom).

Here you could add something like `A work around is to [disable the system sounds](#faq) for the PDF plugin.`

The `[xx](link)` format in markdown is used to create a link to a page, or part of a page. Here we are using only #faq. # means that we want to send to an anchor in the page, and this anchor is called faq.

```suggestion
> [!NOTE]
> When you extract text from the PDF file, the windows error sound may play.
> A work around is to [disable the system sounds](#faq) for the PDF plugin.
```

File: docs/sma/plugin-pdf.md [hunk 2]
```diff
@@ -191,4 +192,8 @@ A: Create an [issue ticket on github](https://github.com/supermemo/SuperMemoAssi
 
 **Q: How to I jump to a specific page number ?**
 
-A: Use <kbd>Ctrl</kbd> + <kbd>G</kbd>.
\ No newline at end of file
+A: Use <kbd>Ctrl</kbd> + <kbd>G</kbd>.
+
+**Q:How do I disable the Windows Sound from playing when I extract from a PDF?**
+
+A: Currently there isn't any workaround, you can disable system sounds by right clicking on the volume icon on your taskbar and setting it to zero.
```
Comment (alexis-) [inline docs/sma/plugin-pdf.md]: I would argue that this is a workaround, since it effectively cancel the issue by another mean that a feature of SMA.

**Review by alexis- — CHANGES_REQUESTED** [review]
Thanks. Can you please review my comments and operate the changes ?

**Review by candywal — COMMENTED** [review]
Made the neccessary changes

File: docs/sma/plugin-pdf.md [hunk 3]
```diff
@@ -166,6 +166,8 @@ Non-hotkeys
 			- Add to selection (Hold ctrl)
 			- Select image (Click image)
 ```
+[!NOTE]
```
Comment (alexis-) [inline docs/sma/plugin-pdf.md]: You're missing > in front of both lines

``` md
> [!NOTE]
> When you extract text from the PDF file, the windows error sound may play. You can [disable it by switching off system sounds](#faq).
```
Also, it is best to remove the ⚠️, this is reserved for really important items, that might break your collection or SMA.

**Review by alexis- — CHANGES_REQUESTED** [review]
Small fix still needed

**Review by candywal — COMMENTED** [review]
:)

**Review by candywal — COMMENTED** [review]
added space

**Review by alexis- — APPROVED** [review]
Fantastic. Thanks

**Pull request merged**